processing/src/main/java/org/apache/carbondata/processing/loading/converter/impl/RowConverterImpl.java

```diff
@@ -119,10 +118,6 @@ public CarbonRow convert(CarbonRow row) throws CarbonDataLoadingException {
             .getTableProperties();
     String spatialProperty = properties.get(CarbonCommonConstants.SPATIAL_INDEX);
     boolean isSpatialColumn = false;
-    Object[] rawData = row.getRawData();
-    if (rawData == null) {
-      rawData = row.getData() == null ? null : row.getData().clone();
```

Comment:
Please revert back this change, somecases rawData will not be set, hence they are setting here before converting the row. 
Now your current code will use null instead of row.getData().clone() in this scenario.

Reply:
I had only added this scenario last time because at that time we wanted rawdata for every bad record action. But now we need this only in case of bad record logger is enable or action is redirect. And in this case, rawdata will be always available.

Reply:
Ok then.